bin/update-captain-formulae

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [[ "$#" -ne 2 ]]; then
+  >&2 echo "Usage: $0 vOLD vNEW"
+  exit 2
+fi
+
+oldVersion=$1
+newVersion=$2
+
+mkdir -p tmp
+
+echo "Getting SHA256 of https://releases.captain.build/${oldVersion}/darwin/aarch64/captain"
+curl "https://releases.captain.build/${oldVersion}/darwin/aarch64/captain" -o ./tmp/captain --fail-with-body --silent
+oldAarch64SHA=$(sha256sum ./tmp/captain | cut -d ' ' -f 1)
+rm ./tmp/captain
+
+echo "Getting SHA256 of https://releases.captain.build/${oldVersion}/darwin/x86_64/captain"
+curl "https://releases.captain.build/${oldVersion}/darwin/x86_64/captain" -o ./tmp/captain --fail-with-body --silent
+oldX8664SHA=$(sha256sum ./tmp/captain | cut -d ' ' -f 1)
+rm ./tmp/captain
+
+echo "Getting SHA256 of https://releases.captain.build/${newVersion}/darwin/aarch64/captain"
+curl "https://releases.captain.build/${newVersion}/darwin/aarch64/captain" -o ./tmp/captain --fail-with-body --silent
+newAarch64SHA=$(sha256sum ./tmp/captain | cut -d ' ' -f 1)
+rm ./tmp/captain
+
+echo "Getting SHA256 of https://releases.captain.build/${newVersion}/darwin/x86_64/captain"
+curl "https://releases.captain.build/${newVersion}/darwin/x86_64/captain" -o ./tmp/captain --fail-with-body --silent
+newX8664SHA=$(sha256sum ./tmp/captain | cut -d ' ' -f 1)
+rm ./tmp/captain
+
+echo "Updating formulae and tests"
+sed -i '' -e "s/version \"${oldVersion}\"/version \"${newVersion}\"/" "./Formula/captain.rb"
+sed -i '' -e "s/version \"${oldVersion}\"/version \"${newVersion}\"/" "./Formula/captain@1.rb"
+sed -i '' -e "s/sha256 \"${oldAarch64SHA}\"/version \"${newAarch64SHA}\"/" "./Formula/captain.rb"
+sed -i '' -e "s/sha256 \"${oldAarch64SHA}\"/version \"${newAarch64SHA}\"/" "./Formula/captain@1.rb"
+sed -i '' -e "s/sha256 \"${oldX8664SHA}\"/version \"${newX8664SHA}\"/" "./Formula/captain.rb"
+sed -i '' -e "s/sha256 \"${oldX8664SHA}\"/version \"${newX8664SHA}\"/" "./Formula/captain@1.rb"
+sed -i '' -e "s/version: \"${oldVersion}\"/version: \"${newVersion}\"/" "./.github/workflows/ci.yml"
```

Comment:
Technically this bit here doesn't really restrict it to Captain only. If we add a script to ABQ, we could make two workflow files instead so it's easier to distinguish in a script.